### DISCUSSION TITLE
New to Quasar, but cant run at all.

### DISCUSSION THREAD

**Discussion by FreakFrog** [quasarframework/quasar#9144] [General - Components / Directives / etc] [answered]
i installed Qusar Cli and created a project then ran quasar dev, and I get this message.
i tried deleting modules/packalge log same issue, my nodejs v is 14.16.1

PS C:\Users\Gigan\Quasar> quasar dev             

 Dev mode.......... spa
 Pkg quasar........ v2.0.0-beta.14
 Pkg @quasar/app... v3.0.0-beta.18
 Pkg webpack....... v5
 Debugging......... enabled       

 Configured browser support (at least 88.19% of global marketshare):
 · Chrome for Android >= 90
 · Firefox for Android >= 87
 · Android >= 90
 · Chrome >= 80
 · Edge >= 87
 · Firefox >= 79
 · iOS >= 11.0-11.2
 · Opera >= 71
 · Safari >= 11.1

 App • Chaining "UI" Webpack config
 App •  WAIT  • Compiling of "UI" in progress...
(node:15196) UnhandledPromiseRejectionWarning:   TypeError: The 'compilation' argument must be an instance of Compilation

  - NormalModule.js:181 Function.getCompilationHooks
    [Quasar]/[webpack]/lib/NormalModule.js:181:10

  - pluginWebpack5.js:37 
    [Quasar]/[vue-loader]/dist/pluginWebpack5.js:37:26

  - Hook.js:14 Hook.CALL_DELEGATE [as _call]
    [Quasar]/[tapable]/lib/Hook.js:14:14

  - Compiler.js:1017 Compiler.newCompilation
    [Quasar]/[app]/[webpack]/lib/Compiler.js:1017:26

  - Compiler.js:1059 
    [Quasar]/[app]/[webpack]/lib/Compiler.js:1059:29

  - Hook.js:18 Hook.CALL_ASYNC_DELEGATE [as _callAsync]
    [Quasar]/[tapable]/lib/Hook.js:18:14

  - Compiler.js:1054 Compiler.compile
    [Quasar]/[app]/[webpack]/lib/Compiler.js:1054:28

  - Watching.js:134 
    [Quasar]/[app]/[webpack]/lib/Watching.js:134:19

(Use `node --trace-warnings ...` to show where the warning was created)
(node:15196) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:15196) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

 • Compiling:
 └── UI ██                   9% setup compilation vue-loader-plugin

**Comment by smolinari** [comment, marked as answer]
@FreakFrog - Are you using NPM? If yes. Try using Yarn please. 

Scott

**Reply by FreakFrog** [reply]
Dev mode.......... spa
 Pkg quasar........ v2.0.0-beta.14
 Pkg @quasar/app... v3.0.0-beta.18
 Pkg webpack....... v5
 Debugging......... enabled

 Configured browser support (at least 88.19% of global marketshare):
 · Chrome for Android >= 90
 · Firefox for Android >= 87
 · Android >= 90
 · Chrome >= 80
 · Edge >= 87
 · Firefox >= 79
 · iOS >= 11.0-11.2
 · Opera >= 71
 · Safari >= 11.1

(node:10888) UnhandledPromiseRejectionWarning:   Error: vue-loader requires @vue/compiler-sfc to be present in the dependency tree.

  - index.js:8 Object.<anonymous>
    [vue-loader-npm-16.2.0-5e70aca9ef-823096da5b.zip]/[vue-loader]/dist/index.js:8:11

  - loader.js:1063 Module._compile
    internal/modules/cjs/loader.js:1063:30

  - loader.js:1092 Object.Module._extensions..js
    internal/modules/cjs/loader.js:1092:10

  - loader.js:928 Module.load
    internal/modules/cjs/loader.js:928:32

  - .pnp.js:17251 Function.external_module_.Module._load
    C:/Users/Gigan/Quasar/.pnp.js:17251:14

  - loader.js:952 Module.require
    internal/modules/cjs/loader.js:952:19

  - helpers.js:88 require
    internal/modules/cjs/helpers.js:88:18

  - create-chain.js:5 Object.<anonymous>
    [@quasar-app-npm-3.0.0-beta.18-6e9a263184-8b6f42eb20.zip]/[@quasar]/app/lib/webpack/create-chain.js:5:29

  - loader.js:1063 Module._compile
    internal/modules/cjs/loader.js:1063:30

  - loader.js:1092 Object.Module._extensions..js
    internal/modules/cjs/loader.js:1092:10


(Use `node --trace-warnings ...` to show where the warning was created)
(node:10888) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:10888) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Reply by smolinari** [reply]
Sorry, forgot to say, remove any lock files and the `node_modules` folder and try installing again.

Scott

**Reply by FreakFrog** [reply]
i started new one and selected Yarn instead of npm in the cli options during the creation ,it worked thank you for the help
